event_model/tests/test_run_router.py

```diff
@@ -216,11 +216,10 @@ def test_run_router_streams(tmp_path):
     )
     docs.append(("start", start_doc))
     stream_resource_doc, compose_stream_datum = compose_stream_resource(
-        spec="TIFF_STREAM",
+        mimetype="application/x-hdf5",
```

Comment:
```suggestion
        mimetype="image/tiff",
```

nit but it'd be nice to preserve previous type to show migration path